SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -3929,6 +3945,29 @@ class CSolver {
    */
   inline virtual unsigned long GetnDOFsGlobal(void) const { return 0; }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
+   */
+  inline virtual void SetTauWall_WF(CGeometry *geometry,
+                                    CSolver** solver_container,
+                                    CConfig* config) { }
+
+  /*!
+   * \brief A virtual member.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
+   */
+  inline virtual void SetNuTilde_WF(CGeometry *geometry,
+                                    CSolver **solver_container,
```

Comment:
These two do not need to be virtual I believe, they are part of the preprocessing and so they can be just private methods of their respective solvers.
Ideally abstract interfaces (like CSolver) should have as few public methods as possible, to make clear what needs to be implemented to define a "solver".

Reply:
yes, changed it.